src/bindings/pygaia/scripts/classification/generate_classification_project.py

```diff
@@ -23,13 +23,22 @@
 from __future__ import with_statement
 import sys
 from os.path import join, abspath
-from optparse import OptionParser
+from argparse import ArgumentParser
 from gaia2 import filedir
 import gaia2.fastyaml as yaml
 
-PROJECT_TEMPLATE = open(join(filedir(), 'classification_project_template.yaml')).read()
 
-def generateProject(groundtruth_file, filelist_file, project_file, datasets_dir, results_dir, seed=None, cluster_mode=False):
+# Use a dict to map essentia versions to the appropriate template so that we
```

Comment:
It'd be good to use this comment to say _why_ we're doing it, instead of _what_ we're doing.
Of course this is a dictionary to map versions. Instead, say why: "Different versions of the music extractor have different layouts, so we have separate project files that reflect the layout of the version that created it", or similar?